src/test/java/org/generationcp/middleware/service/pedigree/PedigreeServiceImplTest.java

```diff
@@ -106,7 +107,12 @@ public void testGetCrossExpansions() {
         final Germplasm generateRandomGermplasm = this.generateRandomGermplasmRecurringMaleParent();
         Set<Integer> set = new HashSet<>();
         set.add(generateRandomGermplasm.getGid());
-        Map<Integer, String> map = this.pedigreeService.getCrossExpansions(set, 5, this.crossExpansionProperties);
-        Assert.assertTrue(map.containsValue("-"));
+        try{
+            this.pedigreeService.getCrossExpansions(set, 5, this.crossExpansionProperties);
```

Comment:
Put Assert.fail after this line:
`this.pedigreeService.getCrossExpansions(set, 5, this.crossExpansionProperties);`